gobblin-core-base/src/test/java/org/apache/gobblin/converter/filter/AvroSchemaFieldRemoverTest.java

```diff
@@ -35,18 +35,30 @@
 
   @Test
   public void testRemoveFields() throws IllegalArgumentException, IOException {
+    // Avro 1.9.2 uses Object as default value and uses internal JacksonUtils.toJson to convert the Object default value
+    // to JsonNode. Enum default value is not supported and hence the conversion fails. Updated the input schema and
+    // removed the non null default value for enum.
     Schema convertedSchema1 = convertSchema("/converter/recursive_schema_1.avsc", "YwchQiH.OjuzrLOtmqLW");
     Schema expectedSchema1 = parseSchema("/converter/recursive_schema_1_converted.avsc");
-    Assert.assertEquals(convertedSchema1, expectedSchema1);
+    Assert.assertEquals(convertedSchema1.toString(), expectedSchema1.toString());
```

Comment:
Looks like we are restricting the condition for equality of two Avro schemas. Why is this change needed after your changes to .avsc files? Also, what does this mean in production pipelines that uses the Schema#equals() checks?